app/helpers/application_helper.rb

```diff
@@ -1381,4 +1381,8 @@ def remote_function(options)
   end
 
   attr_reader :big_iframe
+
+  def appliance_name
+    MiqServer.my_server.settings.fetch_path(:server, :appliance_name)
```

Comment:
Can we use `MiqServer.my_server.name` instead of looking it up in the settings?  In other words, what is the difference between the name attribute in the miq_servers table and the appliance_name in the settings?  @dclarizio @gtanzillo - can you comment on this